src/main/java/com/example/bookstoreapp/exception/CustomGlobalExceptionHandler.java

```diff
@@ -55,10 +55,20 @@ public ResponseEntity<Object> handleRuntimeException(RuntimeException ex, WebReq
         Map<String, Object> body = new LinkedHashMap<>();
         body.put(TIMESTAMP_KEY, LocalDateTime.now());
         body.put(STATUS_KEY, HttpStatus.INTERNAL_SERVER_ERROR);
-        body.put(ERRORS_KEY, "An unexpected error has occurred: " + ex.getMessage());
+        body.put(ERRORS_KEY, ex.getMessage());
         return new ResponseEntity<>(body, HttpStatus.INTERNAL_SERVER_ERROR);
```

Comment:
please replace `HttpStatus.INTERNAL_SERVER_ERROR`

Reply:
> please replace `HttpStatus.INTERNAL_SERVER_ERROR`

Don't really understand this. With what? It is for RuntimeException. I believe it should be INTERNAL_SERVER_ERROR. Or maybe I don't get smth